lib/esyBuildRuntime.sh

```diff
@@ -10,37 +14,45 @@ FG_RESET='\033[0m'
 ESY__SANDBOX_COMMAND=""
 
 if [[ "$esy__platform" == "darwin" ]]; then
-  ESY__SANDBOX_COMMAND="sandbox-exec -f $cur__target_dir/_esy_sandbox.sb"
+  ESY__SANDBOX_COMMAND="sandbox-exec -f $cur__target_dir/_esy/sandbox.sb"
 fi
 
-esy-prepare-install-tree () {
-  mkdir -p $cur__target_dir
-  mkdir -p          \
-    $cur__install   \
-    $cur__lib       \
-    $cur__bin       \
-    $cur__sbin      \
-    $cur__man       \
-    $cur__doc       \
-    $cur__share     \
+_esy-prepare-build-env () {
+
+  rm -rf $cur__install
```

Comment:
Where was this before? I see it (before) *moving* from `cur__install` to `esy_cur__install`, but where was it deleting the old `cur__install` before?